### ISSUE TITLE
[BUG] - JSON parsing fails to error if the field name contains "constructor"

### ISSUE THREAD

**Issue by tomijarvi** [cars10/elasticvue#60]
**Description**
Elasticvue fails in the data fetching from Elastic if some of the field names contain the term ~~"construction"~~ "constructor" e.g. ~~"project_construction"~~ "project_constructor". I investigated this error a bit and the error message is "Object contains forbidden constructor property" which leads to JSON-bigint component (https://github.com/sidorares/json-bigint/blob/390482a8b6b460f98c61c3b65915dbd91fc8e7b2/lib/parse.js#L360). Its regex doesn't take a note that the field can contain also ~~"construction"~~ "constructor" in the field name.

**Steps To Reproduce**
1. Add field e.g. ~~"project_construction"~~ "project_constructor" into elastic index data
2. Open elasticvue
3. Click Indices on menu
4. Open index's "Show info"
5.  It pop up the error: "Network error: failed to fetch."

**Screenshots**
Error message:
![image](https://user-images.githubusercontent.com/1843169/127001719-d4bc1e60-fbae-46cd-a193-e45d276941a7.png)

**Environment (please include the following information):**
* Elasticsearch version: 7.12.0
* Operating system: MacOS
* Browser + version: Chrome 91.0.4472.164
* Elasticvue version: 0.35.0
* How are you running elasticvue? (docker, browser extension, web app): Browser extension

**Additional context**
Thanks for this awesome tool! 😃

**Comment by cars10** [comment]
Hey @tomijarvi , thank you for the detailed bug report! Unfortunately i am not able to reproduce this error. Are you sure that a key named "construction" is the issue here? I have tried to create some data with this key and everything works. I tried the following:

```bash
curl -X POST "http://localhost:9200/construction/_doc" -H 'Content-Type: application/json' -d '{
  "project_construction": true,
  "construction": 5,
  "construction_ids": [
    1,
    2,
    3
  ],
  "constructions": {
    "first_construction": "done",
    "construction_two": "pending"
  }
}'

```

This document has multiple fields that contain the word "construction", but i can display the document and also the index information just fine. Can you provide some concrete example data?

**Comment by tomijarvi** [comment]
Thanks for the quick answer @cars10. I wrote a typo it's "constructor", NOT "construction". Sorry!

**Comment by cars10** [comment]
Thanks! I can reproduce it with `constructor`.

**Comment by cars10** [comment]
Okay, having fields named `constructor` or `__proto__` causes all kinds of issues. I will try to work around that, but i will have to rename the fields in the frontend, for example like so:
![image](https://user-images.githubusercontent.com/4118628/127021356-e6b5f0e6-fd13-454c-96e5-a989bd0b2df8.png)

**Comment by cars10** [comment]
Fixed in `0.36.0`
